### PR TITLE
[ADD] Add base_community module

### PR DESCRIPTION
**Pull request by YannickB**
First module to be reviewed. Simple module which add a community configuration menu.

### CODE REVIEW

**Comment by YannickB** [comment]
Just saw the errors with travis, do we have any way to exclude TODO and unreviewed directories from the tests?

File: __TODO__/crowdfunding/crowdfunding.py [hunk 1]
```diff
@@ -52,32 +41,52 @@ def _get_funded(self, cr, uid, ids, prop, unknow_none, context=None):
             res[campaign.id] = ''
         return res
 
-
     _columns = {
         'crowdfunding': fields.boolean('Launch a crowdfunding campaign?'),
         'crowdfunding_description': fields.text('Crowdfunding description'),
-        'crowdfunding_currency_mode': fields.selection([('one','Only one currency'),('all','All currency')], 'Currency mode', required=True, help="Specify the currency mode, if you select one then you'll get the currency when a goal for one currency if attein but you'll lose all other currency. If you select all, you must atteign each goal for each currency but you'll win all the currency"),
+        'crowdfunding_currency_mode': fields.selection(
```
Comment (PCatinean) [inline __TODO__/crowdfunding/crowdfunding.py:47]: Would placing the selection values on the class be a better approach as far as extensibility is concerned?

File: __TODO__/crowdfunding/crowdfunding.py [hunk 2]
```diff
@@ -21,25 +21,14 @@
...
-import base64
+from openerp.osv import fields, osv
 
-import logging
-#_logger = logging.getLogger(__name__)
 
-
-
-class crowdfunding_campaign(osv.AbstractModel):
+class CrowdfundingCampaign(osv.AbstractModel):
```
Comment (PCatinean) [inline __TODO__/crowdfunding/crowdfunding.py]: Isn't osv deprecated since 6.1?

File: __TODO__/crowdfunding/crowdfunding.py [hunk 3]
```diff
@@ -128,24 +153,27 @@ class crowdfunding_transaction(osv.osv):
     _columns = {
         'model': fields.char('Related Document Model', size=128, select=1),
         'res_id': fields.integer('Related Document ID', select=1),
-        'crowfunding_transfer_anyway': fields.boolean('The receiver will be able to get the currency even without atteigning the goal'),
+        'crowfunding_transfer_anyway': fields.boolean(
+            'The receiver will be able to get the currency'
+            ' even without atteigning the goal'
+        ),
         'state': fields.selection([
```
Comment (PCatinean) [inline __TODO__/crowdfunding/crowdfunding.py:160]: Same as previous comment regarding selections

**Comment by YannickB** [comment]
I replaced osv.osv by osv.Model, I think you're right osv.osv is deprecated. osv itself is still used I think, since I'm not using the new api yet.

Also I'm sorry since I pep8 most of the code to pass the travis test, the diff of the PR became quite a mess. I'm just asking here to review the module base_community, not the whole repo of course.

**Comment by gdgellatly** [comment]
orm rather than osv Yannick.  osv itself is deprecated.  Regards the selection fields, a matter of taste and selection_add is now in v8, but either put them in the class, or make them a function.

**Comment by YannickB** [comment]
Ok thank you for your comments.

This PR became quite a mess and we can even automatically merge it anymore, I should never make the Travis corrections on this branch. I'll close it and merge the Travis correction on another PR. When Travis will be green, I'll make another PR to publish base_community.

**Pull request closed without merging**